doc/UsersGuide/BuildingRunningTesting/tutorials/tutorial1.inc

```diff
@@ -0,0 +1,454 @@
+
+**Weather Phenomena:** Numerous tornado and wind reports (6/15) and hail reports (6/16)
+
+   * `Storm Prediction Center (SPC) Storm Report for 20190615 <https://www.spc.noaa.gov/climo/reports/190615_rpts.html>`_
+   * `Storm Prediction Center (SPC) Storm Report for 20190616 <https://www.spc.noaa.gov/climo/reports/190616_rpts.html>`_
+
+.. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/Tutorial/IndySevereWeather18z.gif
+   :alt: Radar animation of severe weather over Indianapolis on June 15, 2019 starting at 18z. The animation shows areas of heavy rain and tornado reports moving from west to east over Indianapolis. 
+
+   *Severe Weather Over Indianapolis Starting at 18z*
+
+Data
+-------
+
+On :srw-wiki:`Level 1 <Supported-Platforms-and-Compilers>` systems, users can find data for the Indianapolis Severe Weather Forecast in the usual input model data locations (see :numref:`Section %s <DataLocations>` for a list). The data can also be downloaded from the `UFS SRW Application Data Bucket <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html>`_. 
+
+   * FV3GFS data for the first forecast (``control``) is located at: 
+   
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/FV3GFS/grib2/2019061518/
+
+   * HRRR and RAP data for the second forecast (``test_expt``) is located at: 
+      
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/HRRR/2019061518/
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/RAP/2019061518/
+
+Load the Workflow
+--------------------
+
+To load the workflow environment, source the lmod-setup file and load the workflow conda environment by running:
+
+.. include:: ../../doc-snippets/load-env.rst
+
+After loading the workflow, users should follow the instructions printed to the console. Usually, the instructions will tell the user to run |activate|. For example, a user on Hera with permissions on the ``nems`` project may issue the following commands to load the workflow (replacing ``User.Name`` with their actual username):
+
+.. code-block:: console
+   
+   source /scratch1/NCEPDEV/nems/User.Name/ufs-srweather-app/etc/lmod-setup.sh hera
+   module use /scratch1/NCEPDEV/nems/User.Name/ufs-srweather-app/modulefiles
+   module load wflow_hera
+   conda activate srw_app
+
+Configuration
+-------------------------
+
+.. include:: ../../doc-snippets/expt-conf-intro.rst
+
+Experiment 1: Control
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Edit the configuration file (``config.yaml``) to include the variables and values in the sample configuration excerpts below. 
+
+.. include:: ../../doc-snippets/file-edit-hint.rst
+
+Start in the ``user:`` section and change the ``MACHINE`` and ``ACCOUNT`` variables. For example, when running on a personal MacOS device, users might set:
+
+.. code-block:: console
+
+   user:
+      RUN_ENVIR: community
+      MACHINE: macos
```

Comment:
Given that MacOS support has been dropped for this release, this should be changed to a supported machine.